docs/products/flink/overview.rst

```diff
@@ -0,0 +1,97 @@
+Service overview
+================
+
+This article gives you an overview of the service plans and features available in Aiven for Apache Flink beta.
+
+You can run Aiven for Apache Flink on AWS or GCP cloud environments. Various deployment options are available, ranging from fully managed access over the public internet to subnet deployments under your own cloud account. All services and infrastructure are single-tenant and dedicated solely to a single customer.
```

Comment:
"AWS or GCP cloud environments" -> "AWS or GCP cloud regions"

Reply:
I'm not sure about this one - I'd prefer to stick with "environments" (or potentially change it to "platforms"), as that's what AWS and GCP are...